### ISSUE TITLE
Important keword argument for negbin contains unicode symbol

### ISSUE THREAD

**Issue by galenlynch** [JuliaStats/GLM.jl#300]
To use `negbin` with an initial value for the shape parameter requires the use of the keyword argument called `initialθ`. Unicode symbols are a pain to type in many programming environments (for example emacs, vim, etc...). Therefore, I think they should not be allowed in the names of keyword arguments. The fact that I must resort to copying and pasting "θ" to even describe this issue demonstrates the problem.

**Comment by nalimilan** [comment]
Makes sense. In general Julia always includes a pure-ASCII variant for methods which use Unicode characters. Here we could allow both names and internally use the one which has been specified.
